### PR TITLE
[ticket/12632] Add twig.debug and twig.auto_reload in config.yml

### PR DESCRIPTION
**Pull request by Nicofuma**
https://tracker.phpbb.com/browse/PHPBB3-12632
https://tracker.phpbb.com/browse/PHPBB3-13206

PHPBB3-13206
PHPBB3-12632

### CODE REVIEW

**Comment by Nicofuma** [comment]
rebased

File: phpBB/phpbb/di/extension/core.php [hunk 1]
```diff
@@ -68,6 +68,19 @@ public function load(array $configs, ContainerBuilder $container)
 			}
 		}
 
+		// Set the Twig options if defined in the environment
+		$definition = $container->getDefinition('template.twig.environment');
+		$twig_environment_options = $definition->getArgument(6);
```
Comment (marc1706) [inline phpBB/phpbb/di/extension/core.php]: Should probably not use a magic number?

Comment (Nicofuma) [inline phpBB/phpbb/di/extension/core.php]: I don't think a constant will have any sense (and where do you want to define it?)

Comment (marc1706) [inline phpBB/phpbb/di/extension/core.php]: At least add an explanation what 6 means.

**Pull request closed without merging**